acceptance/openstack/swr/v2/organizations_test.go

```diff
@@ -98,15 +95,15 @@ func TestOrganizationPermissionsWorkflow(t *testing.T) {
 		Username: auth.Username,
 		Auth:     1,
```

Comment:
updating for auth value not working still equals 3

Reply:
done